src/client/components/PasswordInput.tsx

```diff
@@ -36,7 +35,6 @@ const textInputBorderStyle = (error?: string) => css`
   bottom: 0px;
   width: 100%;
   height: ${height.inputMedium}px;
-  margin-bottom: ${space[3]}px;
```

Comment:
The password eye on the password input field has an absolute position which takes this margin into account. By removing the bottom margin of this element we have more control over the spacing around this form field without having to worry about putting the eye out of  alignment